scripts/start-restcomm.sh

```diff
@@ -30,14 +30,16 @@ startRestcomm() {
 		'standalone'*)
 			# start restcomm on standalone mode
 			chmod +x $RESTCOMM_HOME/bin/standalone.sh
-			echo ' Telestax RestComm started running on standalone mode. Terminal session: restcomm.'
-			echo "Using IP Address: $BIND_ADDRESS"
 			if [[ "$RUN_DOCKER" == "true" || "$RUN_DOCKER" == "TRUE" ]]; then
+     			echo 'Telestax RestComm started running on standalone mode.'
+	    		echo "Using IP Address: $BIND_ADDRESS"
 			    #change to second runlevel.
 			    runsvchdir second >/dev/null
 				$RESTCOMM_HOME/bin/standalone.sh -b $bind_address "${ExtraOpts}"
 			else
 				tmux new -s restcomm -d "$RESTCOMM_HOME/bin/standalone.sh -b $bind_address ${ExtraOpts}"
+     			echo 'Telestax RestComm started running on standalone mode. Terminal session: restcomm.'
+	    		echo "Using IP Address: $BIND_ADDRESS"
```

Comment:
@agafox cool - then it seems like a quick & easy fix here to increase the indentation. Makes it more readable and that's always a bonus in shell scripts!!  :D ;)